Makefile

```
@@ -9,6 +9,10 @@ clean:
 specs:
 	crystal spec -p --debug --release --error-trace -Dpreview_mt
 
+.PHONY: specs_no_mt
+specs_no_mt:
+	crystal spec -p --debug --release --error-trace -Dpreview_mt
```

but it still has `-Dpreview_mt` flag :slightly_smiling_face:
```suggestion
	crystal spec -p --debug --release --error-trace
```

---

This ^ :sweat_smile: